src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -0,0 +1,114 @@
+import { bolt } from '@api/slack';
+import * as scoresUtils from '@utils/scores';
+
+import { triggerSlackScores } from './slackScores';
+
+describe('slackScores Tests', function () {
+  let getIssueEventsForTeamSpy, postMessageSpy;
+  beforeAll(() => {
+    getIssueEventsForTeamSpy = jest.spyOn(scoresUtils, 'getIssueEventsForTeam');
+    postMessageSpy = jest.spyOn(bolt.client.chat, 'postMessage');
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should handle case when no issues are returned', async () => {
+    getIssueEventsForTeamSpy.mockReturnValue([]);
+    await triggerSlackScores(null, null);
+    expect(postMessageSpy).toHaveBeenCalledWith({
+      blocks: [
+        {
+          text: {
+            emoji: true,
+            text: '🗓️ Weekly GitHub Response Times by Team 🗓️',
+            type: 'plain_text',
+          },
+          type: 'header',
+        },
+        {
+          text: {
+            text: `\`\`\`┌────────────────────────────────────────────────┐
+| Team                          │ % on Time      |
+├────────────────────────────────────────────────┤
+| ospo                          | -   (0/0)      |
```

Comment:
nit: results that are equivalent in all other sorting metric (just % on time, right?) should be sorted alphabetically. This will make tests more deterministic, and the message itself easier to parse for users.